Bottom: 4806632d132d2c93d750c14f86f382b417631c38
Top:    4c3540ae88c6f4a1b7afc49a6a0e5030833db3f5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-23 19:16:24 -0500

Refresh of update-gitlab-to-10-1

---
diff --git a/roles/gitlab-omnibus/tasks/main.yml b/roles/gitlab-omnibus/tasks/main.yml
index 6d915a22..dabd77ea 100644
--- a/roles/gitlab-omnibus/tasks/main.yml
+++ b/roles/gitlab-omnibus/tasks/main.yml
@@ -30,20 +30,20 @@
 # *****************************************************************************
 # backup script part
 
-# - name: Assemble dir for backup scripts
-#   file:
-#     path: /usr/libexec/bacula/backup-scripts
-#     state: directory
+- name: Assemble dir for backup scripts
+  file:
+    path: /usr/libexec/bacula/backup-scripts
+    state: directory
 
-# - name: before_backup script part
-#   template:
-#     src: before_backup.j2
-#     dest: /usr/libexec/bacula/backup-scripts/55.before_backup.gitlab
+- name: before_backup script part
+  template:
+    src: before_backup.j2
+    dest: /usr/libexec/bacula/backup-scripts/55.before_backup.gitlab
 
-# - name: after_backup script part
-#   template:
-#     src: after_backup.j2
-#     dest: /usr/libexec/bacula/backup-scripts/55.after_backup.gitlab
+- name: after_backup script part
+  template:
+    src: after_backup.j2
+    dest: /usr/libexec/bacula/backup-scripts/55.after_backup.gitlab
 
 # *****************************************************************************
 # update the Docker restore config
